tests/foreman/maintain/test_upgrade.py

```diff
@@ -118,15 +124,18 @@ def test_negative_pre_upgrade_tuning_profile_check(request, custom_host):
     :expectedresults: Pre-upgrade check fails.
     """
     profile = request.node.callspec.id
+    rhel_major = custom_host.os_version.major
     sat_version = ".".join(settings.server.version.release.split('.')[0:2])
-    # Register to CDN for RHEL8 repos, download and enable last y stream's ohsnap repos,
+    # Register to CDN for RHEL repos, download and enable last y stream's ohsnap repos,
     # and enable the satellite module and install it on the host
     custom_host.register_to_cdn()
     last_y_stream = last_y_stream_version(
         SATELLITE_VERSION if sat_version == 'stream' else sat_version
     )
     custom_host.download_repofile(product='satellite', release=last_y_stream)
-    custom_host.execute('dnf -y module enable satellite:el8 && dnf -y install satellite')
+    custom_host.execute(
+        f'dnf -y module enable satellite:el{rhel_major} && dnf -y install satellite'
+    )
```

Comment:
Can we add a check here and split this module enable part as this is only for EL8?